siman/core/cluster_batch_script.py

```diff
@@ -806,9 +806,13 @@ def write_footer(cl, set_mod = '', run_tool_flag = True,
 
 
 
-            run_command(cl, option = option, name = cl.id[0]+'.'+cl.id[1]+'.100'+name_mod+'.fitted', 
-                parrallel_run_command = parrallel_run_command, write = True, mpi = mpi, corenum = corenum, f = f)
-
+            # this is tmp solution 
+            if cl.calculator == 'qe':
```

Comment:
Is there any difference here?

Reply:
there were different naming for qe fitted output. fixed now